tests/qos/test_qos_sai.py

```diff
@@ -38,6 +38,15 @@
 
 PTF_PORT_MAPPING_MODE = 'use_orig_interface'
 
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exception(rand_one_dut_hostname, loganalyzer):
+    """ignore the syslog ERR syncd0#syncd: [03:00.0] brcm_sai_set_switch_attribute:1920 updating switch mac addr failed with error -2"""
+    ignore_regex = [
+            ".*ERR syncd[0-9]*#syncd.*brcm_sai*"
```

Comment:
please make this msg specific to ignore only the switch mac addr failed.

Reply:
> please make this msg specific to ignore only the switch mac addr failed.
thanks for pointing out, done